release-notes.md

```diff
@@ -1,5 +1,31 @@
 # Release notes
 
+## May 20, 2019
+
+The following updates were made in this release:
+
+#### iOS
+
+iOS Core 2.2.2 is now available!
+
+* Fixed the issue where SSL connection was not kept alive.
+* Added support for ISO 8601 timestamp for Rules Engine token replacement, you can use the placeholder `~timestampz` 
+* Fixed the bug where `ACPCore.getSdkIdentities` didn't return correct value for custom analytics id.
+* Added support for migarting Privacy status from v4 to v5, if it was manually set with v4 SDK.
+* Identity: Fixed a bug where the callback was not called on getIdentifiers if there were no custom identifiers synced before
```

Comment:
Can you add the Identity notes in a separate section? This will help developers which installed pod versions manually.
Kevin also improved the text for these notes, see below.

\#### iOS Identity 2.0.3:
* Fixed issue where device locale was not properly read when creating network requests from Identity extension.
* Fixed issue where the callback was not called on `ACPIdentity.getIdentifiers` if no custom identifiers where synced previously.
* Fixed issue where the Analytics visitor identifier (VID) was not included in the `ACPIdentity.appendToUrl` callback value.